checks-superstaq/checks_superstaq/requirements.py

```diff
@@ -200,10 +200,14 @@ def _check_package_versions(
 def _get_latest_version(package: str) -> str:
     base_package = package.split("[")[0]  # remove options: package_name[options] --> package_name
     pypi_url = f"https://pypi.org/pypi/{base_package}/json"
-    pypi_version = json.loads(urllib.request.urlopen(pypi_url).read().decode())["info"]["version"]
+    try:
+        package_data = urllib.request.urlopen(pypi_url).read().decode()
+        pypi_version = json.loads(package_data)["info"]["version"]
+    except urllib.error.URLError:
+        return checks_superstaq.__version__
```

Comment:
This is really a temporary fix for the fact that the `requirements.py` check will fail until `checks-superstaq` is on PyPI.  Not unreasonable since we take the `max` of the PyPI and local package versions below anyways.